### PR TITLE
Update docs Java tracker version 2.0.0

### PR DESCRIPTION
**Pull request by matus-tomlein**
Bump version of the tracker and update the cookie jar docs.

### CODE REVIEW

File: docs/collecting-data/collecting-from-own-applications/java-tracker/configuring-how-events-are-sent/index.md [hunk 1]
```diff
@@ -246,19 +246,18 @@ The default thread pool uses non-daemon threads. To stop the threads and shut do
...
-
-Any custom `OkHttp` `CookieJar` could be used instead.
+As described [here](/docs/collecting-data/collecting-from-own-applications/java-tracker/tracking-specific-client-side-properties/index.md), the event collector sets a third-party cookie. This cookie is extracted during event processing (enrichment phase) into the `network_userid` property, the server-side user identifier. To persist this cookie across requests, use the `OkHttpClientWithCookieJarAdapter` when creating your `BatchEmitter` or `Tracker`. Note that the `OkHttpClientWithCookieJarAdapter` uses an in-memory cookie jar, so the cookies, and `network_userid`, will not persist when it goes out of memory.
```
Comment (mscwilson) [inline docs/collecting-data/collecting-from-own-applications/java-tracker/configuring-how-events-are-sent/index.md:249]: add an admonition saying when the `OkHttpClientWithCookieJarAdapter` was released?

File: docs/collecting-data/collecting-from-own-applications/java-tracker/configuring-how-events-are-sent/index.md [hunk 2]
```diff
@@ -246,19 +246,18 @@ The default thread pool uses non-daemon threads. To stop the threads and shut do
...
 
 The simplest implementation looks like this:
 ```java
 Tracker tracker = Snowplow.createTracker(
                   new TrackerConfiguration("namespace", "appId"),
-                  new NetworkConfiguration("http://collector").cookieJar(new CollectorCookieJar()));
+                  new NetworkConfiguration("http://collector").httpClientAdapter(new OkHttpClientWithCookieJarAdapter()));
```
Comment (mscwilson) [inline docs/collecting-data/collecting-from-own-applications/java-tracker/configuring-how-events-are-sent/index.md:255]: the `OkHttpClientWithCookieJarAdapter` has a required property of collector url

**Review by mscwilson — APPROVED** [review]
LGTM!

File: docs/collecting-data/collecting-from-own-applications/java-tracker/configuring-how-events-are-sent/index.md (hunk 2 above)
Comment (matus-tomlein) [reply docs/collecting-data/collecting-from-own-applications/java-tracker/configuring-how-events-are-sent/index.md]: Good catch! Have fixed that.

**Pull request merged**